source/whats-new.txt

```diff
@@ -37,6 +38,29 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.6:
+
+What's New in 5.6
+-----------------
+
+The {+driver-short+} v5.6 release includes the following features:
+
+- The driver now supports Node.js v20.
+
+- The driver can return a cursor as the response to a server command by
+  calling the ``runCursorCommand()`` method. To learn more about this feature,
+  see the `runCursorCommand API documentation <{+api+}/5.6/classes/Db.html#command>`__.
+
+- Support for specifying time series collection creation options
+  `bucketMaxSpanSeconds <{+api+}/interfaces/TimeSeriesCollectionOptions.html#bucketMaxSpanSeconds>`__
+  and
+  `bucketRoundingSeconds <{+api}/interfaces/TimeSeriesCollectionOptions.html#bucketRoundingSeconds>`__.
```

Comment:
```suggestion
- Support for the following options for creating time series collections:
   -`bucketMaxSpanSeconds <{+api+}/interfaces/TimeSeriesCollectionOptions.html#bucketMaxSpanSeconds>`__
   - `bucketRoundingSeconds <{+api}/interfaces/TimeSeriesCollectionOptions.html#bucketRoundingSeconds>`__
```

Reply:
I was debating between the two, but didn't see much advantage in listing them separately since there's no accompanying text. Do you think it's necessary to present as a list?

Reply:
It's not necessary, but as it is the phrase "specifying time series collection creation options" is long and confusing. I thought it could be helpful to break the phrasing up a bit to make it more clear

Reply:
Discussed on Slack